modules/azure/monitoring_action_group/main.tf

```diff
@@ -0,0 +1,41 @@
+terraform {
+  required_version = "~> 1.3"
+
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.40"
+    }
+  }
+
+  backend "azurerm" {}
+}
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_monitor_action_group" "action_group" {
+  name                = var.name
+  resource_group_name = var.resource_group_name
+  short_name          = var.short_name
+  enabled             = var.enabled
+
+  dynamic "email_receiver" {
+    for_each = var.email_receiver == null ? [] : [1]
+    content {
+      name                    = "Email"
+      email_address           = var.email_receiver
```

Comment:
```suggestion
      email_address           = each.value
```

Reply:
`each.value` should be tested.
It could also be `email_receiver.value` i'm not 100% sure.